### PR TITLE
database seed.js file code review

### PR DESCRIPTION
**Pull request by HourglassMagi**
Please review this pull request for me. let me know if there are any significant changes I need to make, including but not limited to adhering to style guide rules. Thank you.

### CODE REVIEW

**Comment by HourglassMagi** [comment]
[PR Laurent's database seed branch seed.js file.](https://trello.com/c/W8RkSwUK/36-pr-laurents-database-seed-branch-seedjs-file)

File: database/seed.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+  while(step <= steps) {
+      let instructions = {};
+      instructions.step = `Step ${step}:`;
+      instructions.title = faker.lorem.slug();
+      instructions.photo = faker.image.food();
+      instructions.instruction = faker.lorem.paragraph();
+      step++;
+      recipe.push(instructions)
+    }
```
Comment (CareyL96) [inline database/seed.js]: looks like this indentation should not be indented

File: database/seed.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+const faker = require('faker');
+const client = require('./index.js');
+
+const insertData = () => {
+}
+
+const createFakeData = () => {
+  for(let i = 0; i <= 100; i++) {
+  const steps = Math.floor(Math.random() * 15) + 4;
```
Comment (careyl96) [inline database/seed.js]: missing indentation in this for loop

**Review by careyl96 — CHANGES_REQUESTED** [review]
fix indentation issues, add a space after the while loop (I also think it should be a for loop, not a while loop; while loops are generally used to repeat a section of code an unknown number of times), and consider renaming the createFakeData function to insertFakeData.

**Comment by HourglassMagi** [comment]
Hey @CareyL96 ,
Thanks for the feedback. I made the requested changes. I decided to keep the while loop because I don't know how many steps there will be for a given recipe. They are being randomly generated between 4 and 15 which is why I decided to user a while loop instead of a for loop. Let me know if you see any other changes I need to make. Thanks.

**Review by careyl96 — DISMISSED** [review]
Missing indentation at the while loop, jsonObj, and text variable initializations since they're all nested within the for loop but you can fix that in the next PR. Consider changing steps to numSteps instead. Looks good otherwise!

**Pull request merged**